docs/releasing/make-release.sh

```diff
@@ -56,18 +56,20 @@ skaffold build --file-output="artifacts.json" --profile "${PROFILE}" \
 skaffold config unset local-cluster
 
 # render manifests
-for service in "frontend contacts userservice balancereader ledgerwriter transactionhistory loadgenerator"; do
-  skaffold render --build-artifacts="artifacts.json" --profile "${PROFILE}" \
-                  --module="${service}" > "${REPO_PREFIX}/${RELEASE_DIR}/${service}.yaml"
+for moduleDashed in frontend contacts userservice balance-reader ledger-writer transaction-history loadgenerator; do
+  module=`echo ${moduleDashed} | tr -d '-'`
+  cp "${SCRIPT_DIR}/header.txt" "${REPO_ROOT}/${RELEASE_DIR}/${moduleDashed}.yaml"
+  skaffold render --build-artifacts="artifacts.json" --profile "${PROFILE}" --namespace "default" \
+                  --module="${module}" >> "${REPO_ROOT}/${RELEASE_DIR}/${moduleDashed}.yaml"
 done
 
 # update version in manifests
-find "${REPO_ROOT}/${RELEASE_DIR}" -name '*.yaml' -exec sed -i -e "s'value: \"dev\"'value: \"${NEW_VERSION}\"'g" {} \;
-rm "${REPO_ROOT}/${RELEASE_DIR}/*-e"
+find "${REPO_ROOT}/${RELEASE_DIR}" -name '*.yaml' -exec sed -i -e "s'value: dev'value: ${NEW_VERSION}'g" {} \;
+rm ${REPO_ROOT}/${RELEASE_DIR}/*-e
 
 # update version in terraform scripts
 sed -i -e "s@sync_branch  = .*@sync_branch  = \"${NEW_VERSION}\"@g" ${REPO_ROOT}/iac/tf-anthos-gke/terraform.tfvars
-rm "${REPO_ROOT}/iac/tf-anthos-gke/terraform.tfvars-e"
+rm ${REPO_ROOT}/iac/tf-anthos-gke/terraform.tfvars-e
```

Comment:
**Question:** Are we removing this to ensure that it doesn't get committed? I wonder if it makes more sense to use `.gitignore`.

Reply:
Yes that's residual from `sed`. Sure, it _could_ be added to the `.gitignore` but my point here is that I don't want them lingering on my machine in the first place, so I'm cleaning them up here :)